flow-model-generator/modules/flow-model-generator-ls-extension/src/main/java/io/ballerina/flowmodelgenerator/extension/ExpressionEditorService.java

```diff
@@ -61,6 +69,65 @@ public Class<?> getRemoteInterface() {
         return null;
     }
 
+    @JsonRequest
+    public CompletableFuture<VisibleVariableTypesResponse> visibleVariableTypes(VisibleVariableTypeRequest request) {
```

Comment:
I think we should improve the function name to reflect the functionality. Add the java docs as well.